python/sdk/tests/test_build.py

```diff
@@ -108,14 +108,21 @@ def test_download(self, unwrap):
 
     @pytest.mark.order(-2)
     def test_delete_test(self, unwrap):
-        unwrap(self.build.delete_test)(self.build, test_id=pytest.test_id)
+        unwrap(self.build.delete_test)(self.build, test_id=pytest.test_id, purge=False)
         res = unwrap(self.detect.get_test)(self.detect, test_id=pytest.test_id)
         pytest.expected_test['tombstoned'] = res['tombstoned']
 
         diffs = check_dict_items(pytest.expected_test, res)
         assert not diffs, json.dumps(diffs, indent=2)
         assert parse(res['tombstoned']) <= datetime.utcnow() + timedelta(minutes=1)
 
+    def test_purge_test(self, unwrap):
```

Comment:
i think you probably need to mark this one as `@pytest.mark.order(-2)` and the one above as `@pytest.mark.order(-3)` to make them run correctly